guidelines/terms/22/perimeter.html

```diff
@@ -2,6 +2,6 @@
 <dd class="new">
 	<p class="change">New</p>	
    <p>continuous line forming the boundary of a shape not including shared pixels, or the <a>minimum bounding box</a>, whichever is shortest.</p>
-  <aside class="example"><p>The perimeter calculation for a rectangle is 2<em>h</em>+2<em>w</em> -4, where <em>h</em> is the height and <em>w</em> is the width and the corners are not counted twice. The perimeter of a circle is 2𝜋<em>r</em>.</p></aside>
+  <aside class="example"><p>The perimeter calculation for a 2 CSS pixel perimeter around a rectangle is 4<em>h</em>+4<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. For a 2 CSS pixel perimeter around a circle it is 4𝜋<em>r</em>.</p></aside>
```

Comment:
This note is wrong. A 2px border around the perimeter isn't 2*width + 2*height. You're missing the corners in that. Instead its 2 * (width+2) + 2 * (height+2).  And for a circle it's not 4𝜋*radius. It's 2𝜋*(radius+1) + 2𝜋*(radius+2).

Reply:
In my opinion, it makes no sense to count the actual pixels for a rectangle and therefore subtract the corner pixels, but for a circle take the theoretical pixels and subtract nothing. The actual pixels for a circle can deviate considerably from 4*PI*radius, especially for small circles.